pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -21,10 +24,37 @@ def __init__(
         validate_data: bool = True,
         control_columns: Optional[List[str]] = None,
         adstock_max_lag: int = 4,
+        yearly_seasonality: Optional[int] = None,
         **kwargs,
     ) -> None:
+        """Media Mix Model with delayed adstock and logistic saturation class (see [1]_).
+
+        Parameters
+        ----------
+        data : pd.DataFrame
+            Training data.
+        target_column : str
+            Column name of the target variable.
+        date_column : str
+            Column name of the date variable.
+        channel_columns : List[str]
+            Column names of the media channel variables.
+        validate_data : bool, optional
+            Wether to validate the data upon initialization, by default True
+        control_columns : Optional[List[str]], optional
+            Column names of control variables to be added as additional regressors, by default None
+        adstock_max_lag : int, optional
+            Number of lags to consider in the adstock transformation, by default 4
+        yearly_seasonality : Optional[int], optional
+            Number of Fourier modes to model yearly seasonality, by default None
```

Comment:
Add "." in the end (?)

Reply:
fixed in https://github.com/pymc-labs/pymc-marketing/pull/242/commits/dc5fe9418cd89425083d02f963fed63f1dcd1c92